test/user/donations.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Module dependencies.
+ */
+import mongoose from 'mongoose';
+import jwt from 'jsonwebtoken';
+import supertest from 'supertest';
+import chai from 'chai';
+import app from '../../server';
+
+const {
+  expect
+} = chai;
+const server = supertest(app);
+const signupUrl = '/api/v1/auth/signup';
+let token; // store token for user authentication
+const expiredToken = jwt.sign({
+  _id: 'h123h1h2hhhhhhs'
+}, 'expired', {
+  expiresIn: '0.001s'
+});
+const User = mongoose.model('User');
+
+// delete all records in User model
+User.collection.drop();
+
+const userDetails = {
+  name: 'Oluwadunsin Oyebiyi',
+  password: 'password',
```

Comment:
Where is the test that actually asserts for the feature we have worked on?

Reply:
Please scroll down to view the tests

Reply:
Do you have any test for "users should be able to view a "game log" of past games"?